reference/constraints/Choice.rst

```diff
@@ -279,6 +279,34 @@ you can pass the class name and the method as an array.
             }
         }
 
+Supplying the Choices from an Array Constant
+--------------------------------------------
+
+You can also directly provide an array constant to the ``choices`` option in the annotation::
+
+    // src/AppBundle/Entity/Author.php
+    namespace AppBundle\Entity;
+
+    class Author
+    {
+        const GENRES = ['fiction', 'non-fiction'];
```

Comment:
I would prefer sth like `action` and `comedy`

Reply:
yep, i agree, but i've chosen these one for the coherence with the example above
would you mind that i change the other example as well ?